backend/cmd/api/main.go

```diff
@@ -329,6 +330,8 @@ func main() {
 
 	r.POST("/public", publicGraphqlHandler(loaders))
 
+	r.GET("/versionz", version.GinHandler)
```

Comment:
Versionz?

Reply:
Note the /versionz basically comes from https://stackoverflow.com/questions/43380939/where-does-the-convention-of-using-healthz-for-application-health-checks-come-f